compact/forwardHcal_standalone.xml

```diff
@@ -0,0 +1,12 @@
+<lccdd>
+
+  <display>
+  </display>
+  <define>
+  </define>
+
+  <documentation level="10">
+    ### Hcal configuration
+  </documentation>
+  <include ref="hcal/hcal_longforward.xml"/>
+</lccdd>
```

Comment:
We should not include this file. The entry points should be handled through the configurations.